src/components/select/simple-select/simple-select.test.js

```diff
@@ -996,4 +1011,14 @@ context("Tests for Simple Select component", () => {
       }
     );
   });
+
+  describe("check height of Select list when opened", () => {
+    it("with long option text, should not cut off any text", () => {
+      CypressMountWithProviders(<SimpleSelectWithLongWrappingTextComponent />);
+
+      selectText().click();
+      selectListWrapper().should("be.visible");
+      selectListWrapper().should("have.css", "height", "152px");
```

Comment:
describe("check height of Select list when opened", () => {
    it("should not cut off any text with long option text", () => {
      CypressMountWithProviders(<SimpleSelectWithLongWrappingTextComponent />);
      selectText().click();
      selectListWrapper().should("have.css", "height", "152px").and("be.visible");
    });
  });